### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by PxlPioneer** [jeremylvln/Shulker#114] [General]
Hello Jeremy,

I was wondering if you had a Discord where I could get in touch with you?

**Comment by jeremylvln** [comment]
Hi @PxlPioneer ,

If you encounter any issue or would want to submit a feature request, I would rather prefer that you use GitHub directly. It would help me keep track of everything.

However, if you really need to contact me privately, send me an email or a Twitter PM (see my profile for the links). I prefer to keep my Discord for my friends, at least for now :)

Thanks!
